rest-api-spec/src/main/resources/rest-api-spec/test/search/40_indices_boost.yaml

```diff
@@ -0,0 +1,171 @@
+setup:
+  - do:
+      indices.create:
+          index:  test_1
+  - do:
+      indices.create:
+          index:  test_2
+
+  - do:
+      indices.put_alias:
+        index: test_1
+        name: alias_1
+
+  - do:
+      indices.put_alias:
+        index: test_2
+        name: alias_2
+
+  - do:
+      index:
+          index:  test_1
+          type:   test
+          id:     1
+          body:   { foo: bar }
+
+  - do:
+      index:
+          index:  test_2
+          type:   test
+          id:     1
+          body:   { foo: bar }
+
+  - do:
+      indices.refresh:
+        index: [test_1, test_2]
+
+---
+"Indices boost using object":
```

Comment:
we need to add a skip section to every test relying on warnings, or the runners not supporting the "warnings" feature will break when trying to run this test.